The 140 types don't compile, while 139 does - GLTFExporter.

<!--

* Please fill out this template with all the relevant information so we can
  understand what's going on and fix the issue. We appreciate bugs filed and PRs
  submitted!

* You can get the installed version of an NPM package by running `npm ls <insert package name>` in your terminal.

-->

-   `three` version: r140.2
-   `node` version: v16.13.2
-   `npm` version: 8.1.2

### Problem description:

I'm upgrading [model-viewer](https://github.com/google/model-viewer/pull/3411) from three.js r139 to r140.2; this is working just fine if I leave @types/three at 0.139.0. However, if I also update the types to 0.140.0, I get the following build error:
```
node_modules/@types/three/examples/jsm/exporters/GLTFExporter.d.ts:94:5 - error TS7010: 'setPlugins', which lacks return-type annotation, implicitly has an 'any' return type.

94     setPlugins(plugins: GLTFExporterPlugin[]);
```

### Suggested solution:

It's clearly a regression, and oddly it's a regression in the types themselves rather than due to the three.js upgrade, since the 139 types are working for both old and new three.js. Hopefully the errant PR shouldn't be too hard to find.

--- a/types/three/examples/jsm/exporters/GLTFExporter.d.ts
+++ b/types/three/examples/jsm/exporters/GLTFExporter.d.ts
@@ -91,7 +91,7 @@ export class GLTFExporter {
 export class GLTFWriter {
     constructor();
 
-    setPlugins(plugins: GLTFExporterPlugin[]);
+    setPlugins(plugins: GLTFExporterPlugin[]): void;
 
     /**
      * Parse scenes and generate GLTF output

